ui/components/multichain-accounts/multichain-account-menu/multichain-account-menu.tsx

```diff
@@ -143,13 +151,20 @@ export const MultichainAccountMenu = ({
       </Box>
       <Popover
         className="multichain-account-cell-popover-menu"
-        isOpen={isPopoverOpen}
+        isOpen={isOpen}
         position={PopoverPosition.LeftStart}
         referenceElement={popoverRef.current}
         matchWidth={false}
         borderRadius={BorderRadius.LG}
+        isPortal
```

Comment:
isPortal renders the menu outside of the main DOM at the end of the body allowing for it to not effect scrolling. [Docs](https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-popover--docs)

<img width="1048" height="251" alt="Image" src="https://github.com/user-attachments/assets/0b6d3c53-d025-4773-89e4-c1fb0d5f722b" />